packages/ember-routing/lib/system/router.js

```diff
@@ -411,8 +411,8 @@ const EmberRouter = EmberObject.extend(Evented, {
     return this.transitionTo(...arguments).method('replace');
   },
 
-  generate() {
-    let url = this._routerMicrolib.generate(...arguments);
+  generate(...args) {
+    let url = this._routerMicrolib.generate(...args);
```

Comment:
Why do we need this change?  As you can see [here](http://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&targets=&browsers=&builtIns=false&code=let%20obj%20%3D%20%7B%0A%20%20generate(...args)%20%7B%0A%20%20%20%20let%20url%20%3D%20this._routerMicrolib.generate(...args)%3B%0A%20%20%20%20return%20this.location.formatURL(url)%3B%0A%20%20%7D%0A%7D%0A%0Alet%20obj2%20%3D%20%7B%0A%20%20generate()%20%7B%0A%20%20%20%20let%20url%20%3D%20this._routerMicrolib.generate(...arguments)%3B%0A%20%20%20%20return%20this.location.formatURL(url)%3B%0A%20%20%7D%0A%7D) the transpiled output is the same before and after.

Seems fine to make the change, but I was just curious what motivated it...